config/initializers/airbrake.rb

```diff
@@ -1,7 +1,12 @@
 # This file is overwritten on deploy
-#
+
 Airbrake.configure do |config|
+  config.api_key = ENV["ERRBIT_API_KEY"]
+  config.host    = ENV["ERRBIT_HOST"]
+  config.environment_name = ENV["ERRBIT_ENV"]
+  config.secure  = ENV["ERRBIT_API_KEY"].present?
+
   # Adding "production" to the development environments causes Airbrake not
   # to attempt to send notifications.
-  config.development_environments << "production"
+  config.development_environments << "production" unless ENV["ERRBIT_API_KEY"].present?
```

Comment:
> My guess would be that it's to prevent Airbrake from sending exceptions even if the environment is set to production on a developer machine.

This is my understanding too. @alext is this correct (I saw you added it [here](https://github.com/alphagov/router-api/blame/master/config/initializers/airbrake.rb))?

> I wonder whether we should be checking for RUNNING_ON_HEROKU as well as ERRBIT_API_KEY?

I think if you have this ENV var set it's pretty clear that you are expecting to communicate with Airbrake. Heroku is out of scope in this file, we would assume too much by adding it here imho.